fmf/base.py

```diff
@@ -548,7 +548,11 @@ def node(reference):
         # FIXME Should be able to remove .cache if required
         return found_node
 
-    def get_raw_data_node(self):
+    def __get_raw_data_node(self):
+        """
+        Get tuple of node where raw_data are stored and list if keys
+        for current node inside raw data
```

Comment:
I don't understand this comment

Reply:
there is typo: "list if keys" -> "list of keys"

Reply:
Plus one for better clarifying the method meaning.